modules/c++/math.linear/unittests/test_mx.cpp

```diff
@@ -81,6 +81,38 @@ TEST_CASE(testScaleMultiplyMxN)
     TEST_ASSERT_EQ(B, C);
 }
 
+TEST_CASE(testNegateMxN)
+{
+    Matrix3x3 A = identityMatrix<3, double>();
+    Matrix3x3 B = -A;
+    Matrix3x3 C = -B;
+    TEST_ASSERT_EQ(A, C);
```

Comment:
Technically if the negation operator did nothing, this unit test would still pass